### ISSUE TITLE
Problems with training

### ISSUE THREAD

**Issue by Szakulli07** [switchablenorms/DeepFashion2#87]
Hello, this is not really issue with dataset but I hope mayby you can help me.

I try to train decent image retrival model on your dataset. However I can not get anything at least near satisfying.

With resnet18 pretrained on image net without any training I get about 2% on validation set when considering top 10 results.

I've already tried using resnet18, resnet34, resnet50, some VITs all pretrained from image net. AdamW as optimizer with lr 1e-3, 1e-4, 1e-5. Triplet loss with margins varying from 1e-2 to 1. Batch hard mining, hard negative mining, easy positive mining and some other kinds of minings. Batch size depending of model from 400 to 1600. Using only shop images, only customer images, using both, training on only validation set. 

But all that didn't get me more than 5.7% on this set. 

I know I can use some more advanced techniques for pooling, rerankng etc but shouldn't I get at least decent result in this basic setup?

I would be more than grateful for any advice. This is small part of my master thesis but for now it is a lot harder than I supposed.
